_assets/stylesheets/components/breadcrumbs.scss

```diff
@@ -36,13 +36,12 @@
 }
 
 .acc-breadcrumb-home {
-  background-image: image_url('icons/globe.svg');
   background-position: center left;
   background-repeat: no-repeat;
   background-size: 1.6em;
   padding: 0;
   text-indent: -9999px;
-  width: 2.4em;
+  cursor: default;
```

Comment:
@jcbpl In the long-run the <a> empty tag should probably be replaced by a <span> or other element, but for now the cursor change is to prevent the hover state from making it appear as though it's a clickable element to the left of the first breadcrumb.